.vib/node-exporter/goss/node-exporter.yaml

```
@@ -0,0 +1,4 @@
+command:
+  check-node-exporter:
+    exec: /opt/bitnami/node-exporter/bin/node_exporter --help
+    exit-status: 0
```

I think we could replace this check with one running the exporter itself and checking it starts properly.

```suggestion
command:
  check-node-exporter:
    exec: timeout --preserve-status 5 node_exporter
    exit-status: 143
    stdout: ....
```